pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -793,9 +793,9 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 			// repetitive log spam
 			switch {
 			case err == images.ErrImagePullBackOff:
-				klog.V(3).Infof("%v start failed: %v: %s", typeName, err, msg)
+				klog.V(3).Infof("%v %q start failed in pod %q: %v: %s", typeName, spec.container.Name, format.Pod(pod), err, msg)
```

Comment:
the two logs above are using `spec.container`, here you opt in for `spec.container.Name`. Would be nice to make logs consistent unless there is a reason to have it differently

(https://github.com/kubernetes/kubernetes/pull/92325/files#diff-8da51ab012ecbdf2b013ecbcf1c37033R784) and (https://github.com/kubernetes/kubernetes/pull/92325/files#diff-8da51ab012ecbdf2b013ecbcf1c37033R788)

Reply:
Thanks, I updated it.

Better to have more information than less when trying to troubleshoot and error.